common/src/main/java/org/opensearch/ml/common/MLConfig.java

```diff
@@ -27,11 +27,18 @@ public class MLConfig implements ToXContentObject, Writeable {
 
     public static final String TYPE_FIELD = "type";
 
+    public static final String CONFIG_TYPE_FIELD = "config_type";
+
     public static final String CONFIGURATION_FIELD = "configuration";
 
+    public static final String ML_CONFIGURATION_FIELD = "ml_configuration";
+
     public static final String CREATE_TIME_FIELD = "create_time";
     public static final String LAST_UPDATE_TIME_FIELD = "last_update_time";
 
+    public static final String LAST_UPDATED_TIME_FIELD = "last_updated_time";
```

Comment:
Is it possible to reuse the old Field names for CX experience BWC, rather than defining new ones?

Reply:
Using old names and trying to update their types is failing. OpenSearch does not allow conversion between certain types and flat_object is one of them